<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>101</width>
    <height>26</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget"/>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupacketCapture">
    <property name="title">
     <string>packetCapture</string>
    </property>
   </widget>
   <widget class="QMenu" name="menurun">
    <property name="title">
     <string>run</string>
    </property>
    <addaction name="actionstart_2"/>
    <addaction name="actionstop_2"/>
    <addaction name="actionclean_2"/>
   </widget>
   <addaction name="menupacketCapture"/>
   <addaction name="menurun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionrun">
   <property name="text">
    <string>run</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionstop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionclean">
   <property name="text">
    <string>clean</string>
   </property>
  </action>
  <action name="actionstart_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/player.png</normaloff>:/resource/player.png</iconset>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="actionstop_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/stop.png</normaloff>:/resource/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionclean_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/clean.png</normaloff>:/resource/clean.png</iconset>
   </property>
   <property name="text">
    <string>clean</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
